stampy-check.yaml

```diff
@@ -0,0 +1,227 @@
+resources:
+- name: src
+  type: pull-request
+  source:
+    access_token: {{github-access-token}}
+    branch: master
+    private_key: {{github-private-key}}
+    uri: git@github.com:hpcloud/stampy.git
+
+- name: status
+  type: status
+  source:
+    access_token: {{github-access-token}}
+    repo: hpcloud/stampy
+
+- name: src-ci
+  type: git
+  source:
+    branch: master
+    private_key: {{github-private-key}}
+    uri: git@github.com:SUSE/stampy-ci.git
+
+- name: stampy-binary-linux
+  type: s3
+  source:
+    access_key_id: {{s3-access-key}}
+    endpoint: {{s3-endpoint}}
+    secret_access_key: {{s3-secret-key}}
+    bucket: stampy
+    regexp: stampy-(.*)\.g.*linux.amd64\.tgz
```

Comment:
Shouldn't the `gxxxxxx` be part of the version string too (so that we always have different versions)?  That way two parallel PRs would end up with different versions…